packages/frontend-api/src/Resources/config/graphql-types/SettingsDecorator.types.yaml

```diff
@@ -24,3 +24,7 @@ SettingsDecorator:
                 type: String!
                 resolve: "@=query('displayTimezoneQuery')"
                 description: "Timezone that is used for displaying time"
+            heurekaEnabled:
+                type: Boolean!
+                resolve: "@=query('heurekaEnabledQuery')"
+                description: "Returns true if the domain locale is supported and Heureka Shop Certification is enabled on the domain"
```

Comment:
I would simplify the text only to `Returns true if Heureka is available for the current domain`